### DISCUSSION TITLE
After publishing the application, the token becomes unavailable

### DISCUSSION THREAD

**Discussion by fb24m** [community/community#61576] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

During development, my application using github api works, but as soon as I publish it to github pages, the api gives a 401 (Bad Credentials) error, and the settings say that the token is not valid. Why is that?

**Comment by GhostOf0days** [comment]
1. Don't use Github Pages. Instead, use Netlify, Heroku, or Vercel.
2. Assuming it is a simple use case/website/web app, use Netlify.
3. Use [this guide](https://docs.netlify.com/environment-variables/get-started/) on setting environment variables for Netlify.
4. Use [this guide](https://ridbay.medium.com/react-routing-and-netlify-redirects-fd1f00eeee95) to set the _redirects file for Netlify.
